src/annotator/sidebar.coffee

```diff
@@ -47,6 +47,16 @@ module.exports = class Sidebar extends Host
     @element.on 'click', (event) =>
       if !@selectedTargets?.length
         this.hide()
+
+    # Mobile browsers do not register click events on
```

Comment:
I thought that this restriction applied only to iOS but not to Android Chrome. This works either way.